packages/imitation_game/README.md

```diff
@@ -93,11 +93,18 @@ An implementation has to follow these rules:
 A single test run can report multiple numbers.
 
 ## Results
-Date created: 2020-08-17 23:57:16.702500Z
+Date created: 2020-09-03 23:42:33.010291Z
+Flutter 1.22.0-10.0.pre.82 • channel master • https:&#x2F;&#x2F;github.com&#x2F;flutter&#x2F;flutter.git
+Framework • revision 9248fda410 (35 minutes ago) • 2020-09-03 16:08:01 -0700
+Engine • revision 3f05b521b9
+Tools • Dart 2.10.0 (build 2.10.0-86.0.dev)
 
 - smiley
     - flutter
-      - startupTime: 0.561475s
+      - ios_startup_time: 0.691717
+      - adb_memory_total: 45148.0
```

Comment:
Are we sure this is right? Seems a bit surprising... Maybe a quick sanity check in Android Studio's memory profiler? :D